src/server/rpc/procedures/roboscape/roboscape.js

```diff
@@ -34,9 +46,9 @@
 const logger = require('../utils/logger')('roboscape');
 var dgram = require('dgram'),
     server = dgram.createSocket('udp4'),
-    NetworkTopology = require('../../../network-topology'),
-    FORGET_TIME = 120, // forgetting a robot in seconds
-    RESPONSE_TIMEOUT = 200, // waiting for response in milliseconds
+    SocketManager = require('../../../socket-manager'),
```

Comment:
socket manager was re done a while back

Reply:
This must have been updated on the main brach after i cloned a local copy several months ago. I did not make any changes to encrypt. So I think we need to look back at the state of roboscape immediately after the summer camp, which is when I began editing this file.